### DISCUSSION TITLE
C# helpProvider SetHelpString does not support unicode

### DISCUSSION THREAD

**Discussion by thelemontree10** [dotnet/csharplang#4290] [General] [answered]
Hi,

I am using help provider to show help for my control.

I input the help string for helpProvider control. But this does not show the string properly.

I have googled and find nothing possible. Anyone here know how to by pass this ?

I have tried in this post but no success.

https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding?view=net-5.0

Thank you.

Best Regards,

P/S:

My language is Vietnam, which is a unicode font.

Here is my text when progamming: "Chúc mừng năm mới"

Here is the text when shown:

![Capture](https://user-images.githubusercontent.com/13205041/103432490-2cd1c780-4c12-11eb-95c8-6a58788cf038.PNG)

**Comment by CyrusNajmabadi** [comment, marked as answer]
I don't believe this is a roslyn issue.  Maybe a Winforms one?

**Reply by thelemontree10** [reply]
Yes, it is on winform

**Reply by theunrepentantgeek** [reply]
I believe @CyrusNajmabadi  is saying that this is the wrong forum for finding an answer to your issue. This repo is for discussions about the C# language itself, not for issues with libraries or tooling. 

As a best guess, try the WinForms repo:
https://github.com/dotnet/winforms

At worst, they’ll know who to talk with.

**Reply by thelemontree10** [reply]
Thank you. I will close.
